lua/starfall/toolscreen.lua

```diff
@@ -0,0 +1,217 @@
+
+if SERVER then
+
+	util.AddNetworkString("sf_notify_tool_error")
+	hook.Add("StarfallChipError", "StarfallToolscreenError", function(chip)
+		if chip.instance.player.IsPlayer then -- Superuser check
+			net.Start("sf_notify_tool_error")
+			net.Send(chip.instance.player)
+		end
+	end)
```

Comment:
I don't like the networking for when the chip errors being done in here. It should be in starfall_processor.